Main Model/household-procedures.nls

```diff
@@ -180,8 +120,11 @@ to buy-consumption-goods  ; household procedure
 
     ; figure out the transaction
     let the-firm one-of firms-to-try
-
+    
+    if liquidity < 0 [set liquidity 0] ;we kept getting a rounding error where households would have a liquidity of 1.6 x 10 ^ -16 due to a rounding error. This corrects for that.
```

Comment:
This doesn't feel like the right place to take care of this. Where is the rounding error happening? Take care of it there